docs/configure/user-interface.md

```diff
@@ -0,0 +1,392 @@
+```
+
+As an example, you can tell Storybook to use the "dark" theme by modifying [`.storybook/manager.js`](./overview#configure-story-rendering):
+
+```js
+// .storybook/manager.js
+import { addons } from '@storybook/addons';
+import { themes } from '@storybook/theming';
+
+addons.setConfig({
+  theme: themes.dark,
+});
+```
+
+When setting a theme, set a full theme object. The theme is replaced, not combined.
+
+### Theming docs
+
+[Storybook Docs](../writing-docs) uses the same theme system as Storybook’s UI, but is themed independently from the main UI.
+
+Supposing you have a Storybook theme defined for the main UI in [`.storybook/manager.js`](./overview#configure-story-rendering):
+
+```js
+// .storybook/manager.js
+import { addons } from '@storybook/addons';
+// or a custom theme
+import { themes } from '@storybook/theming';
+
+addons.setConfig({
+  theme: themes.dark,
+});
+```
+
+Here's how you'd specify the same theme for docs in [`.storybook/preview.js`](./overview#configure-story-rendering):
+
+```js
+// .storybook/preview.js
+import { themes } from '@storybook/theming';
+
+// or global addParameters
+export const parameters = {
+  docs: {
+    theme: themes.dark,
+  },
+};
+```
+
+Continue to read if you want to learn how to create your theme.
+
+### Create a theme quickstart
+
+The easiest way to customize Storybook is to generate a new theme using the `create()` function from `storybook/theming`. This function includes shorthands for the most common theme variables. Here's how to use it:
+
+First create a new file in `.storybook` called `yourTheme.js`.
+
+Next paste the code below and tweak the variables.
+
+```js
+// yourTheme.js
+import { create } from '@storybook/theming/create';
+
+export default create({
+  base: 'light',
+
+  colorPrimary: 'hotpink',
+  colorSecondary: 'deepskyblue',
+
+  // UI
+  appBg: 'white',
+  appContentBg: 'silver',
+  appBorderColor: 'grey',
+  appBorderRadius: 4,
+
+  // Typography
+  fontBase: '"Open Sans", sans-serif',
+  fontCode: 'monospace',
+
+  // Text colors
+  textColor: 'black',
+  textInverseColor: 'rgba(255,255,255,0.9)',
+
+  // Toolbar default and active colors
+  barTextColor: 'silver',
+  barSelectedColor: 'black',
+  barBg: 'hotpink',
+
+  // Form colors
+  inputBg: 'white',
+  inputBorder: 'silver',
+  inputTextColor: 'black',
+  inputBorderRadius: 4,
+
+  brandTitle: 'My custom storybook',
+  brandUrl: 'https://example.com',
+  brandImage: 'https://placehold.it/350x150',
+});
+```
+
+Finally, import your theme into [`.storybook/manager.js`](./overview#configure-story-rendering) and add it to your Storybook parameters.
+
+```js
+// .storybook/manager.js
+import { addons } from '@storybook/addons';
+import yourTheme from './yourTheme';
+
+addons.setConfig({
+  theme: yourTheme,
+});
+```
+
+The `storybook/theming` package is built using TypeScript, so this should help create a valid theme for typescript users. The types are part of the package itself.
```

Comment:
`@storybook/theming`?

Reply:
That slipped. Also in the the docs it's also referenced that way :P